promscale/installation/kubernetes/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Install Promscale on Kubernetes
+excerpt: Install Promscale on Kubernetes using instrumentation and without using instrumentation
+product: promscale
+keywords: [analytics]
+tags: [learn, prometheus]
+related_pages:
+  - /promscale/:currentVersion:/installation/resource-recomm/
+  - /promscale/:currentVersion:/send-data/
+---
+
+# Install Promscale on Kubernetes
+By default, Promscale does not provide instrumentation to collect telemetry
+from your applications on Kubernetes. You need to install 
+[Prometheus][prometheus-install] to collect metrics and 
+[OpenTelemetry][otel] to collect traces.
+You can install Promscale on Kubernetes with or without instrumentation.
+
+## Install Promscale on Kubernetes without instrumentation
+If you already have Prometheus or OpenTelemetry installed, you can install Promscale
+on Kubernetes using [Helm charts][promscale-install-helm] or 
+[a manifest][promscale-install-k8s-manifest] file.
+
+After you have installed Promscale, migrate your existing Prometheus data into
+Promscale using the [prom-migrator tool][promscale-install-prom-migrator], then
+configure [Prometheus][config-prometheus] and [OpenTelemetry][config-otel] to
+send data to Promscale.
+
+## Install Promscale on Kubernetes with instrumentation
```

Comment:
This section is a duplicate of Tobs PR. Let's have this heading and 2-3 lines of definition on Tobs and link to Tobs about.